packages/twenty-front/src/modules/object-record/cache/utils/getRecordNodeFromRecord.ts

```diff
@@ -65,20 +65,23 @@ export const getRecordNodeFromRecord = <T extends ObjectRecord>({
           return undefined;
         }
 
-        if (Array.isArray(value)) {
-          const objectMetadataItem = objectMetadataItems.find(
-            (objectMetadataItem) => objectMetadataItem.namePlural === fieldName,
+        if (
+          Array.isArray(value) &&
+          field.type !== FieldMetadataType.MultiSelect
+        ) {
+          const item = objectMetadataItems.find(
+            (omi) => omi.namePlural === fieldName,
```

Comment:
we avoid shortnames

Reply:
i had a hard time understanding that stuff because all sub scoped values where named `objectMetadataItem`. They were 3 different `objectMetadataItem` in that function

Reply:
Will rename nevertheless